spec/spec_helper.rb

```diff
@@ -1,6 +1,6 @@
 require 'rspec/its'
 
-Dir['./spec/support/**/*'].each {|f| require f}
+Dir['./support/**/*.rb'].each {|f| require f}
```

Comment:
Do you need to change this? I don't see any support directory under spec.

Reply:
It was leftover from my previous specs, however it's arguably more implicit having `.rb` appended. Happy to remove though.